lib/generators/feature_flag/feature_flag_generator.rb

```diff
@@ -0,0 +1,17 @@
+class FeatureFlagGenerator < Rails::Generators::NamedBase
+  REPLACEMENT_REGEX = /(def run).*?(end.*)\z/m
+
+  def create_feature_flag_config
+    append_file "config/feature_flags.yml", %(  - #{file_name}\n)
+  end
+
+  def create_dus
+    script_name = "add_#{file_name}_feature_flag"
+    generate "data_update", script_name, "--no-spec"
```

Comment:
Not generating a spec seemed sensible here, we'd essentially just be testing that `FeatureFlag.add` works which we do elsewhere.